magic_filter/util.py

```diff
@@ -1,14 +1,14 @@
-from typing import Any
+from typing import Any, Sequence, Set, Union
 
 
-def in_op(a: Any, b: Any) -> bool:
+def in_op(a: Union[Sequence[Any], Set[Any]], b: Any) -> bool:
     try:
         return b in a
     except TypeError:
         return False
```

Comment:
@JrooTJunior,
Is it correct to mute the exception?
E.g. if `a is None` - it would be nice to raise an exception, isn't is?